packages/pubsub/src/subscription.js

```diff
@@ -678,63 +560,40 @@ Subscription.prototype.delete = function(callback) {
  */
 Subscription.prototype.pull = function(options, callback) {
```

Comment:
We don't want pull to be publicly exposed to our users. We want the event emitter on the Subscription object to be the only way users can receive messages.